### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by Enoch-Hsiao**
Once a user logs in, they will be presented with their scenario dashboard UI. 

Used a lot of material UI grids to make the scenario cards look right, any suggestions/improvements and **anything from minimal to drastic UI changes/ideas** are welcomed. Also **comments for _clearer_ naming of components, files, the buttons, and etc.**  are wanted.

Also feel free to pull the branch and test out the UI of the dashboard page and tweak with the CSS. 

In PR:
- Dashboard page /dashboard, dashboard.js
- Scenario card and Add new scenario button are separate components (ScenarioCard.js and AddNewScenarioCard.js)
- Edit button redirects to /editor/:id (id of scenario), the scenario editor page/editor.js
- Data button redirects to /data/:id (id of scenario), the data page/data.js to view student responses
- In both pages, props/data is passed from dashboard.js to editor and data pages, for further reading: 
[pass-props-in-link-react-router](https://stackoverflow.com/questions/30115324/pass-props-in-link-react-router)
- In index.js set primary theme to UMass colors:     
```
primary: {
      main: '#881c1c',
      light: '#F7E7E7',
    },
```

Here's how the dashboard UI looks like with fake data:
Dashboard UI

### CODE REVIEW

**Review by benjaminxu251 — DISMISSED** [review]

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/index.js [hunk 1]
```diff
@@ -3,10 +3,22 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/index.js]: Great!

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      >
+        {scenarios}
+        <AddNewScenarioCard />
+      </Grid>
+      <Box className={classes.copyright}>
+        <Copyright />
+      </Box>
+    </Container>
+  );
+}
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/dashboard.js]: EOL

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  { scenarioName: "Scenario 6", className: "CS6", id: '6'},
+  { scenarioName: "Scenario 7", className: "CS7", id: '7'},
+];
+
+//Passing props with React Router in Material UI: https://stackoverflow.com/questions/30115324/pass-props-in-link-react-router
+export default function Dashboard() {
+  const classes = useStyles();
+
+  //TODO change when Scenario Objects are defined
+  let scenarios = sampleScenariosData.map(data =>
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/dashboard.js:41]: nice use of map

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/index.js [hunk 4]
```diff
@@ -3,10 +3,22 @@ import ReactDOM from 'react-dom';
...
+      main: '#881c1c',
+      light: '#F7E7E7',
+    },
+  },
+});
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <MuiThemeProvider theme={theme}>
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/index.js]: great!

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/data.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    textAlign: 'center'
+  },
+  copyright: {
+    marginTop: theme.spacing(2),
+  }
+}));
+
+export default function Data(props) {
+  const classes = useStyles();
+  console.log(props);
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/data.js]: should this still be here?

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/signup.js [hunk 6]
```diff
@@ -26,8 +26,8 @@ const useStyles = makeStyles((theme) => ({
   },
   submit: {
     marginTop: theme.spacing(2),
-    backgroundColor:  '#881c1c',
-    color: "white"
+    backgroundColor: theme.palette.primary.main,
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/signup.js:29]: nice

**Review by powellsl99 — DISMISSED** [review]
Really great PR! You are making great use of MUI+React, and the code is very well written and readable. My only suggestion is there are a lot of EOL errors git is showing, you may want to look into husky+prettier soon in a different PR which will auto fix that for you before commit. But really nice work!!
Edit: Also dont forget that this needs to be ADA compliant. One of you should look into what that means. It will involve stuff like adding aria to the components

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/data.js (hunk 5 above)
Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/data.js]: No, thanks for pointing this out

**Review by powellsl99 — APPROVED** [review]
LGTM! Thanks for fixing the EOL issues!

**Pull request merged**